app/components/ArtistHero/ArtistHero.module.scss

```diff
@@ -0,0 +1,89 @@
+.container{
+  max-width: 1176px;
+  width: 100%;
+  height: 372px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-between;
+  padding: 24px;
+  background-size: cover;
+  background-position: center;
+  border-radius: 16px;
+}
+
+.h{
+  color: #FFF;
+  font-family: Inter;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+}
+
+.artist{
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-start;
+  gap: 4px;
+}
+
+.artistName{
+  color: #FFF;
+  font-family: Inter;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+
+}
+
+.songsCount{
+  color: #FFF;
+  font-family: Inter;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+}
+
+@media only screen and (max-width: 1023px){
+  .container{
+    max-width: 696px;
+    height: 372px;
```

Comment:
In general, if the style does not change, then it is not necessary to repeat the same style in the media, you are simply duplicating the code.